src/odemis/acq/align/transform.py

```diff
@@ -94,3 +99,315 @@ def CalculateTransform(optical_coordinates, electron_coordinates, skew=False):
         shear_ret = -shear
 
         return (translation_xy_ret[0], translation_xy_ret[1]), (scaling_ret, scaling_ret), theta_ret, scaling_xy_ret, shear_ret
+
+
+class Transform(object):
+    """
+    A Transform class that is useful for geometrical coordinate transformations.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
I'd rather have the arguments explicit. So, I'd suggest to only allow to set rotation, scaling, shear, and translation from init. To set the transformation_matrix and rotation_matrix, the caller has to do it explicitly after init.